Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -406,6 +402,23 @@ TextureCache::TCacheEntryBase* TextureCache::Load(const u32 stage)
 		++iter;
 	}
 
+	if (unconverted_copy != textures.end())
+	{
+		// Perform palette decoding.
+		TCacheEntryBase *entry = unconverted_copy->second;
+		TCacheEntryBase *decoded_entry = AllocateTexture(entry->config);
```

Comment:
Maybe we should add a check if the config only allocate one mipmap layer. It's always true right now, but very fragile.